.github/workflows/rust_ci.yml

```
@@ -0,0 +1,83 @@
+  run:
+    working-directory: backend
+
+jobs:
+  rustfmt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@stable
+      - uses: Swatinem/rust-cache@v2
+        with:
+          working-directory: backend
```

actionlint (1.7.7)

input "working-directory" is not defined in action "Swatinem/rust-cache@v2". available inputs are "cache-all-crates", "cache-bin", "cache-directories", "cache-on-failure", "cache-provider", "cache-targets", "env-vars", "key", "lookup-only", "prefix-key", "save-if", "shared-key", "workspaces"

(action)